src/dev-app/cdk-experimental-listbox/cdk-listbox-demo.css

```diff
@@ -0,0 +1,14 @@
+.example-listbox {
+  list-style-type: none;
+  border: 1px solid black;
+  cursor: default;
+  padding: 0;
+}
+
+.cdk-option-selected {
```

Comment:
Could you change these styles to only target css classes prefixed with `demo-`? It makes debugging the dev-app easier if you can know immediately where the styles are coming from